sparse_strips/vello_hybrid/src/render.rs

```diff
@@ -325,48 +325,19 @@ impl Renderer {
     /// Render to a texture
     pub fn render_to_texture(
         &mut self,
-        device: &Device,
-        queue: &Queue,
         scene: &Scene,
-        texture_view: &TextureView,
-        render_params: &RenderParams,
+        render_pass: &mut RenderPass<'_>,
+        _render_params: &RenderParams,
```

Comment:
If `render_params` isn’t being used, should we consider removing it for now?

Reply:
I think so; looking at `fine`, the clear colour for the blend layers is transparent black:
https://github.com/linebender/vello/blob/fafba255bdf9584dae0e4b7b621e2a0cf2df7281/vello_shaders/shader/fine.wgsl#L977

 That means that if we render into a user-provided `RenderPass`, we never need to know the clear colour, so we can also remove that field.